Tutorial_1b_Moment_calculation.ipynb

```diff
@@ -0,0 +1,860 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    for kk in range(12):
</pre><p>Can you add some explanation for the choice of loops here?</p><p> </p><p>Could you use binary_dilation instead? <a href="https://docs.scipy.org/doc/scipy/reference/generated/scipy.ndimage.binary_dilation.html" target="_blank">https://docs.scipy.org/doc/scipy/reference/generated/scipy.ndimage.binary_dilation.html</a></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Smithsonian/sma-science-tutorials/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>